### PR TITLE
Feature/multiapp

### PR DESCRIPTION
**Pull request by igorpolt**

### CODE REVIEW

File: etc/browser-client.api.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ export interface ClientOptions {
     language?: string;
     loginUrl?: string;
     microphone?: Microphone;
+    projectId: string;
```
Comment (arzga) [inline etc/browser-client.api.md:49]: Is projectId a mandatory parameter? If not, should it be marked as `projectId?`

**Review by arzga — DISMISSED** [review]
Works well based on my tests

File: etc/browser-client.api.md (hunk 1 above)
Comment (igorpolt) [reply etc/browser-client.api.md]: Its optional. Fixed.

**Review by arzga — APPROVED** [review]
LGTM

**Pull request merged**